.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,3 +26,8 @@ Thanks for filling in who performed the code review and the test!
 - [ ] **MAJOR** - when you make incompatible API changes
 - [ ] **MINOR** - when you add functionality in a backwards compatible manner
 - [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions
+
+## Implementation Plan
```

Comment:
Could you motivate a little bit?

Reply:
Sometimes we should do things as running a script on production-database or deploy an Orderform to Susy, this would be a checklist of those things.

Reply:
I think it will serve as a good reminder - if it is not not relevant one can just omit filling it in or delete it from the PR header.